swarm/tools/flow_studio_ui/index.html

```
@@ -323,6 +323,7 @@ <h3 id="shortcuts-modal-title">Keyboard Shortcuts</h3>
   <div class="selftest-step-content" data-uiid="flow_studio.modal.run_detail.body">
     <button class="selftest-modal-close" id="run-detail-close" aria-label="Close run detail modal" data-uiid="flow_studio.modal.run_detail.close">×</button>
     <div id="run-detail-modal-content">
+      <button id="run-detail-rerun" class="hidden" data-uiid="flow_studio.modal.run_detail.rerun">Re-run</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)